src/Element.js

```diff
@@ -136,6 +137,86 @@ export default class Element extends ParentNode {
         });
         return _array;
     }
+
+    /**
+     * Returns the first element that is a descendant of the element on which it is invoked that matches the
+     * specified group of selectors.
+     * @param {String} query CSS query for selecting element
+     * @return {Element} reference to an Element, or null if element was not found
+     */
+    querySelector(query) {
+        let parser = new CssSelectorParser.CssSelectorParser();
+        let rules = parser.parse(query);
+        var result = null;
```

Comment:
It is. let is the same as var, but limited to the block on witch it is declared. The variable is accessible to inner functions, unless you define an other variable inside it